### PR TITLE
[#231] Fix missing descriptions in OpenAPI and json schema during generation

### PR DESCRIPTION
**Pull request by kaniyan**
* This fixes the issue #231

* Also bamm-cli has been extended to accept a preferred language as a parameter.

* Also maven plugin has been extended to accept a preferred language as a parameter.

* bamm-cli and maven plugin documentation have been updated.

### CODE REVIEW

File: documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc [hunk 1]
```diff
@@ -83,12 +83,14 @@ The available options and their meaning can also be seen in the help text of the
                                         |  _--paging-cursor-based, -pc_ : in case there is more than one paging possibility, it has to be cursor based paging|
                                         |  _--paging-offset-based, -po_ : in case there is more than one paging possibility, it has to be offset based paging|
                                         |  _--paging-time-based, -pt_ : in case there is more than one paging possibility, it has to be time based paging|
+                                        |  _--language, -l_ : The language from the model for which a OpenAPI specification should be generated (default: en) |  `bamm aspect AspectModel.ttl to openapi -l de`|
```
Comment (atextor) [inline documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc:86]: _an_ OpenAPI specification

File: documentation/developer-guide/modules/tooling-guide/pages/maven-plugin.adoc [hunk 2]
```diff
@@ -248,6 +249,7 @@ Configuration Properties:
 | `aspectOffsetBasedPaging` | Set the used paging strategy as offset-based paging. | `Boolean` | `false` | {nok}
 | `aspectTimeBasedPaging` | Set the used paging strategy as time-based paging. | `Boolean` | `false` | {nok}
 | `outputFormat` | The format of the resulting OpenAPI Specification. May be either JSON or YAML. | `String` | none | {ok}
+| `language` | The language from the model for which a OpenAPI specification should be generated. | `String` | en | {nok}
```
Comment (atextor) [inline documentation/developer-guide/modules/tooling-guide/pages/maven-plugin.adoc:252]: _an_ OpenAPI specification

**Review by atextor — CHANGES_REQUESTED** [review]

File: documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc (hunk 1 above)
Comment (kaniyan) [reply documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc]: Thanks for the review. I fixed it now.

File: documentation/developer-guide/modules/tooling-guide/pages/maven-plugin.adoc (hunk 2 above)
Comment (kaniyan) [reply documentation/developer-guide/modules/tooling-guide/pages/maven-plugin.adoc]: Fixed

**Review by atextor — APPROVED** [review]

**Pull request merged**